### PR TITLE
VMSS sample update: list the vmss nics, instances and ssh collection string

### PR DESCRIPTION
**Pull request by anuchandy**

### CODE REVIEW

File: azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java [hunk 1]
```diff
@@ -251,6 +258,38 @@ public static void main(String[] args) {
...
+                // List virtual machine scale set instance network interfaces and SSH connection string
+
+                System.out.println("Listing scale set virtual machine instance network interfaces and SSH connection string...");
+                for (VirtualMachineScaleSetVM instance : virtualMachineScaleSet.virtualMachines().list()) {
+                    System.out.println("Scale set virtual machine instance #" + instance.instanceId());
+                    System.out.println(instance.id());
+                    PagedList<VirtualMachineScaleSetNetworkInterface> networkInterfaces = instance.listNetworkInterfaces();
+                    // Pick the first NIC
+                    VirtualMachineScaleSetNetworkInterface networkInterface = networkInterfaces.get(0);
+                    for (VirtualMachineScaleSetNicIpConfiguration ipConfig :networkInterface.ipConfigurations().values()) {
```
Comment (anudeepsharma) [inline azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java]: >:networkInterface [](start = 75, length = 17)

space after :

File: azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java [hunk 2]
```diff
@@ -29,6 +34,8 @@
  * Azure Compute sample for managing virtual machine scale sets -
  *  - Create a virtual machine scale set behind an Internet facing load balancer
  *  - Install Apache Web servers in virtual machines in the virtual machine scale set
+ *  - List the network interfaces associated with the virtual machine scale set
+ *  - List scale set virtual machine instances and SSH collection string
```
Comment (anudeepsharma) [inline azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java]: If this change is for current sample you need to update the individual sample repo as well.

**Comment by anudeepsharma** [comment]
:shipit:

**Comment by anudeepsharma** [comment]
LGTM

File: azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java (hunk 2 above)
Comment (anuchandy) [inline azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachineScaleSet.java]: yes, I have updated the sample repo as well.

**Pull request merged**